server/app/controllers/me-controller.js

```diff
@@ -71,11 +71,14 @@ async function updateMe(req, res) {
     }
     if (user.barman) {
         const schema = BarmanSchema.min(1);
+
         const newUser = req.body.barman;
 
         const { error } = schema.validate(newUser);
         if (error) throw createUserError('BadRequest', error.message);
 
+        if (newUser.active) throw createUserError('BadRequest', 'Active field is not allowed throw PUT /me');
```

Comment:
I've tested:
- If `active` field is present (set to true or false): server throw `BadRequest, Active field is not allowed throw PUT /me` error because active field is compute as a string.
- If `active` field is undefined: server throw a `BadRequest, child \"active\" fails because [\"active\" must be a boolean]` error

So what do you advice @Embraser01 ?

Reply:
With Joi?

Otherwise, I'm ok with this

Reply:
For the part where the field is undefined, it's not really important.
Joi allow you to send your own error so use it :wink:

Reply:
So as you want :) As always, be free to modify it before merge :)